charts/piraeus/values.yaml

```diff
@@ -123,12 +123,15 @@ operator:
     extraVolumes: []
     customLabels: {}
     customAnnotations: {}
-haController:
+piraeus-ha-controller:
   enabled: false
-  image: quay.io/piraeusdatastore/piraeus-ha-controller:v0.3.0
-  affinity: {}
-  tolerations: []
+  image:
+    repository: quay.io/piraeusdatastore/piraeus-ha-controller
+    tag: ""
+  imagePullSecrets: []
+  nameOverride: "ha-controller"
+  fullnameOverride: ""
   resources: {}
-  replicas: 1
-  customLabels: {}
-  customAnnotations: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
```

Comment:
Same as above regarding the values, but in this case we do not even need the repository, I think?

Reply:
yes, I leave there only `resources, affinity and tolerations` now